catalog/MDCCatalog/MDCCatalogTiles.m

```diff
@@ -3044,10 +3047,18 @@ void MDCCatalogDrawPageControlTile(CGRect frame, NSObject<MDCColorScheme> *color
   CGColorSpaceRelease(colorSpace);
 }
 
-static NSString* const PalettesFontName = @"RobotoMono-Regular";
-
 void MDCCatalogDrawPalettesTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme) {
   CGFloat PalettesFontPointSize = [UIScreen mainScreen].bounds.size.width / 320 * 7;
+   
+   UIFont *(^PalettesFont)(void) = ^(void){
```

Comment:
I think this could just be a `UIFont *` that you assign using an if/else.  

```objectivec
 UIFont *font;

  if (isRunningiOS9OrGreater()) {
    font = [UIFont monospacedDigitSystemFontOfSize:12 weight:UIFontWeightRegular];
  } else {
    font = [UIFont systemFontOfSize:12];
    UIFontDescriptor *descriptor = [[font fontDescriptor] fontDescriptorWithSymbolicTraits:UIFontDescriptorTraitMonoSpace];
    font = [UIFont fontWithDescriptor:descriptor size:0.0];
  }
```